src/connectors/pglite.ts

```diff
@@ -0,0 +1,75 @@
+import { PGlite, type PGliteOptions } from "@electric-sql/pglite";
+import type { Connector, Statement } from "../types";
+
+export type ConnectorOptions = PGliteOptions
```

Comment:
I have simplified types for now to avoid duplication and possibly going out of sync